packages/nextjs/components/Footer.tsx

```diff
@@ -9,13 +9,40 @@ export default function Footer() {
   const ethPrice = useAppStore(state => state.ethPriceSlice.ethPrice);
 
   return (
-    <div className="bg-secondary p-4 flex-shrink-0">
-      {ethPrice > 0 && (
-        <div className="btn btn-primary btn-sm font-normal cursor-auto">
-          <CurrencyDollarIcon className="text-black h-4 w-4 mr-0.5" />
-          <span>{ethPrice}</span>
-        </div>
-      )}
+    <div className="navbar bg-secondary min-h-0 p-4">
+      <div className="navbar-start">
+        {ethPrice > 0 && (
+          <div className="btn btn-primary btn-sm font-normal cursor-auto">
+            <CurrencyDollarIcon className="text-black h-4 w-4 mr-0.5" />
+            <span>{ethPrice}</span>
+          </div>
+        )}
+      </div>
+      <div className="navbar-middle">
+        <ul className="menu menu-horizontal px-1">
+          <div className="flex items-center gap-4">
+            <div>
+              <a href="https://github.com/scaffold-eth/se-2" target="_blank" rel="noreferrer">
+                Fork Me
+              </a>
+            </div>
+            <div>
+              Built with 💖 at 🏰{" "}
+              <a href="https://buidlguidl.com/" target="_blank" rel="noreferrer">
+                BuidlGuidl
+              </a>
+            </div>
+            <div>
+              <a href="https://google.com/" target="_blank" rel="noreferrer">
```

Comment:
I think that the link here should be the link to the Telegram group https://t.me/joinchat/KByvmRe5wkR-8F_zz6AjpA

Carlos commented it at the related issue https://github.com/scaffold-eth/se-2/issues/67

Reply:
oh my bad, didn't realize the link there. Updated https://github.com/scaffold-eth/se-2/pull/75/commits/158ecbbfd473e05e5a38e6e2ca5828db988b4643

Thank you Damu!!